ha/core/system_health/const.py

```diff
@@ -31,6 +32,8 @@ class HEALTH_STATUSES(Enum):
     OFFLINE = "offline"
     FAILED = "failed"
     UNKNOWN = "unknown"
+    PENDING = "pending"
```

Comment:
@ajay-paratmandali Are these the health status values really?

Reply:
yes initial status is pending status 

COMPLETE = "complete"
Not sure if this is status so will remove in next patch

Reply:
Updated Rule